en/reference/query-api-reference.html

```diff
@@ -720,12 +720,23 @@ <h3 id="ranking.queryCache">ranking.queryCache</h3>
 <tr><td>Default</td><td>false</td></tr>
 </table>
 <p>
-Turns query cache on or off.
-Query is a two-phase process.
-If the query cache is on, the query is stored on the content nodes
-between the first and second phase,
-saving network bandwidth and also query setup time,
-at the expense of using more memory.
+  Turns query cache on or off. Query is a two-phase process.
+  If the query cache is on, the query is stored on the content nodes between the first and second phase,
+  saving network bandwidth and also query setup time, at the expense of using more memory.
+  It only affects the protocol phase two,
+  see <a href="../performance/caches-in-vespa.html">caches in Vespa</a>.
+  It does not cache the result, it just saves resources by not forwarding the query twice
+  (one for the first protocol phase which is find the best k documents from all nodes,
+  to the second phase which is to fill summary data
+  and potentially ranking features listed in summary-features in the ranking profile).
+</p>
+<p>
+  The <a href="schema-reference.html#summary-features">summary-features</a>
+  are re-calculated but this setting avoids sending the query down once more.
+  There is little downside of using but won't save much.
```

Comment:
```suggestion
  There is little downside of using it and it can save resources and latency in cases where the query tree and query ranking features (e.g tensors used in ranking) are large.  
```